lib/compress/zstd_lazy.c

```diff
@@ -1140,12 +1140,12 @@ FORCE_INLINE_TEMPLATE void ZSTD_row_prefetch(U32 const* hashTable, U16 const* ta
  */
```

Comment:
The relation between `iLimit` (end of prefetching range) and `iend` (end of input buffer, not present in the function's interface) could be explained in code comment, especially if it justifies the way `iLimit` is used in the code below.